careercrumbs/careercrumbs-client
# Ask

#### What's this PR do?
Plenty. 
1. Adds functionality to create questions and see them in an index page.
2. Adds a reducer test.
3. Brings package.json and webpack configs up to root level.
#### Where should the reviewer start?
Jump in where we left off together - the commit where we added the action creator and reducer. 
#### How should this be manually tested?
Go to `/ask`, create a couple questions, then visit `/questions` to see them displayed.
#### Any background context you want to provide?
Chai was a pain in the ass until I figured out a `deep.equal` was required. There's a lot in here, look through and ask questions if you have any. Might be better ways to do some of this.
#### Screenshots (if appropriate)
<img width="544" alt="screen shot 2015-11-29 at 8 19 06 am" src="https://cloud.githubusercontent.com/assets/3621728/11459859/623eeedc-969d-11e5-9861-2a6ccb00eb4c.png">
Updated, with q's from server:
<img width="606" alt="screen shot 2015-12-02 at 8 28 54 am" src="https://cloud.githubusercontent.com/assets/3621728/11534964/db4119d0-98ce-11e5-83fb-0d93ab721c51.png">

#### What gif best describes this PR or how it makes you feel?
![msty](https://cloud.githubusercontent.com/assets/3621728/11459854/4e9dce84-969d-11e5-940c-0fd3db4a1a6b.gif)
